app/models/application_form.rb

```diff
@@ -119,6 +119,10 @@ def any_offers?
     application_choices.map.any?(&:offer?)
   end
 
+  def application_not_sent?
+    application_choices.map.all?(&:application_not_sent?)
```

Comment:
I don't think you need the map:

```suggestion
    application_choices.all?(&:application_not_sent?)
```